geotrellis-sentinelhub/src/test/scala/org/openeo/geotrellissentinelhub/PyramidFactoryTest.scala

```diff
@@ -527,7 +529,9 @@ class PyramidFactoryTest {
       from_date = ISO_OFFSET_DATE_TIME format date,
       to_date = ISO_OFFSET_DATE_TIME format date,
       band_names = Seq("HV", "HH").asJava,
-      metadata_properties = Collections.singletonMap("polarization", Collections.singletonMap("eq", "DH"))
+      metadata_properties = util.Collections.emptyMap[String, util.Map[String, Any]],
+      // gives no results with "polarization" filter?
+      //metadata_properties = Collections.singletonMap("polarization", Collections.singletonMap("eq", "DH")),
```

Comment:
Possibly. These comments can be removed.

Reply:
Removed